src/app-root.ts

```diff
@@ -95,6 +100,16 @@ export class AppRoot extends LitElement {
     }
   }
 
+  private collectionChanged(e: Event) {
+    e.preventDefault();
+    this.withinCollection = this.baseCollectionField.value;
+    this.collectionBrowser.withinCollection = this.withinCollection;
+
+    if ((this.currentPage ?? 1) > 1) {
+      this.collectionBrowser.goToPage(this.currentPage ?? 1);
```

Comment:
SUGGESTION: Use meaningful temp variable instead of calc'ing `this.currentPage ?? 1` twice?